h2o-hadoop/tests/python/pyunit_read_invalid_file.py

```diff
@@ -11,7 +11,9 @@ def read_invalid_file():
       hdfs_path = 'hdfs:///user/jenkins/tests/invalid'
       h2o.import_file(hdfs_path)
       assert False, "Read of file, which does not exists was sucessfull. This is impossible"
-  except ValueError:
+  except ValueError as ve:
+      print(ve)
+      assert str(ve) == "ImportFiles of 'hdfs:///user/jenkins/tests/invalid' failed on [u\"Path does not exist: 'hdfs:/user/jenkins/tests/invalid'\"]"
       pass
```

Comment:
The test was failing because of type error on line 289 in h2o.py. `paths` is a list and it cannot be concatenated with string without explicit conversion.